es/index.js

```diff
@@ -140,6 +157,23 @@ function fixConfig(axios, config) {
  *   return retryCount * 1000;
  * }});
  *
+ * // Custom retry which checks Retry-After header against limit
+ * axiosRetry(axios, { retryDelay : (retryCount, error) => {
+ *   if (error.response) {
+ *     const retry_after = error.response.headers["retry-after"];
+ *     if (retry_after) {
+ *       // check if retry is less than 5 seconds
```

Comment:
Ideally I'd like to compare to `axios.timeout` but I couldn't figure out how to do that.